aemedge/blocks/side-nav/side-nav.css

```diff
@@ -29,6 +31,8 @@ aside {
     & :is(a, .text) {
       color: var(--nav-text-color);
       font-size: var(--nav-links-font-size);
+      font-family: var(--navlinks-font-family);
```

Comment:
Is that needed, is there a difference between this font and the body font? Where is the car defined?